doc/bufexplorer.txt

```diff
@@ -510,7 +510,7 @@ CHANGE LOG                                              *bufexplorer-changelog*
     - Dave Eggum has made some 'significant' updates to this latest
       version:
       * Added BufExplorerGetAltBuf() global function to be used in the
-        user�s rulerformat.
+        users rulerformat.
```

Comment:
Oops.  This wasn't me, this was GitHub's inline editor silently removing what it considered to be an invalid character.

Aren't encodings fun?